pkg/otel/types.go

```diff
@@ -42,6 +44,11 @@ type LoggingConfig struct {
 	Receivers []string
 }
 
+type TraceConfig struct {
+	Enabled   bool
+	Receivers []string
```

Comment:
Probably don't need an array of receivers in the `TraceConfig` since these will be static for now